src/renderer/components/database-item.jsx

```diff
@@ -169,7 +169,10 @@ export default class DatabaseItem extends Component {
 
     return (
       <div>
-        <span style={style} className="item" onContextMenu={this.onContextMenu}>
+        <span
+          style={style}
+          className={`item item${dbObjectType}`}
```

Comment:
I like having a divider here between the class and the metatype, similar to how other libraries do it, eg. `item__${dbObjectType}` or `item-${dbObjectType}`.